Automattic/pocket-casts-ios
# Add Sentry Dsyms upload Steps to CI Release Cycle

Fixes #341 

| Depends on https://github.com/Automattic/pocket-casts-ios/pull/359 |
|:---:|

Adds the DSYM upload steps to the release cycle. 

## To test

1. Set the environment variable `SENTRY_AUTH_TOKEN` with an appropriate auth token
2. Set up a test lane like:
```
lane :upload_test do |options|
    sentry_check_cli_installed
    configure_code_signing

    gym(
      scheme: 'pocketcasts',
      include_bitcode: false,
      include_symbols: true,
      clean: true,
      export_options: { method: 'app-store' }
    )

    symbols_upload
end
```
3. run your test lane `bundle exec fastlane upload_test`
4. Verify the Dsyms made it to sentry. 

## Checklist
